azurerm/internal/services/network/resource_arm_application_gateway.go

```diff
@@ -662,7 +662,7 @@ func resourceArmApplicationGateway() *schema.Resource {
 						"capacity": {
 							Type:         schema.TypeInt,
 							Optional:     true,
-							ValidateFunc: validation.IntBetween(1, 32),
+							ValidateFunc: validation.IntBetween(1, 125),
```

Comment:
Could we add a check to the create/update functions to ensure the 32 limit is respected when not using a V2 SKU?

Reply:
Maybe adding CustomizeDiff function is better. I've updated code. Please have a look. Thanks.